src/services/syntaxServices.js

```diff
@@ -0,0 +1,19 @@
+services.factory('syntaxServices', [function() {
+  
+	function validateNifSyntax(nif) {
```

Comment:
Yes, this was done before discussion about nifs; I'm redoing all the service; maybe tomorrow it will be done